tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -1034,6 +1034,7 @@ final case class TestingServer(
       filename: String,
       maxRetries: Int = 4,
   ): Future[List[l.CodeLens]] = {
+    Debug.printEnclosing(filename)
```

Comment:
Can you explain to me what does this thing do?

Reply:
just prints that this method has been invoked together with the filename. 

This is where things like `2023.10.06 10:30:46 INFO  tests.BaseLspSuite#initialize ` come from.